delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/Files.scala

```diff
@@ -245,6 +245,37 @@ final class Files(
     } yield res
   }.span("registerFile")
 
+  def updateRegisteredFile(
+      id: FileId,
+      storageId: Option[IdSegment],
+      rev: Int,
+      entity: HttpEntity,
```

Comment:
Why do we have a http entity here ?
It should be a path like the `registerFile` no ?